tests/unit/ui/components/TestRecommendedBadge.js

```diff
@@ -26,4 +30,12 @@ describe(__filename, () => {
 
     expect(root).toHaveClassName(className);
   });
+
+  it('Calls event.stopPropagation after clicking on the link', () => {
```

Comment:
Since specs start with `it` when you read them, we typically don't capitalize the first word.